README.md

```diff
@@ -12,18 +12,21 @@ We currently support a variety of features.
 * Auto-completion based on context and type via the Lean server
 * Error messages / diagnostics
 * Customizable Unicode input support (e.g. type `\la`+<kbd>tab</kbd> to input `λ`)
-* Info view window to show goal, tactic state, and error messages (<kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>enter</kbd> for local goal view or <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>alt</kbd>+<kbd>enter</kbd> for all messages)
+* Info view window to show goal, tactic state, and error messages:
+  - click <img src="media/display-goal-light.svg"> or hit <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>enter</kbd> for local goal view, or
```

Comment:
BTW, you can't use SVGs in the readme of a vscode extension.  I didn't know this either.  I've now changed the SVGs to PNGs: https://github.com/leanprover/vscode-lean/commit/e3b5a822291a2ebc5d4358dab41e4a602c211c90

Reply:
Ah, OK. VS Code gave me a warning about those, but I wasn't sure what it was referring to.